src/Lykke.Sdk/LykkeAppOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using JetBrains.Annotations;
+
+namespace Lykke.Sdk
+{
+    [PublicAPI]
+    public class LykkeAppOptions
+    {
+        public string AppName { get; set; }
```

Comment:
Seems that we could use PlatformServices.Default.Application.ApplicationVersion, at least as default value

Reply:
https://github.com/aspnet/Announcements/issues/237
will use System.AppDomain.CurrentDomain.SetupInformation.ApplicationName as default